scrapbook-tutorial/src/Comment.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+
+function CommentBody(props) {
+  const rows = props.characterData.map((row, index) => {
+    return (
+      <div className="comment" key={index}>
```

Comment:
It might be nice to move this bit into a separate component.  That way you have 1 that contains the list and 1 that defines the individual component that keeps appearing in the list.  I think this is more of a comment list than a comment body.

Reply:
Separated this out now